hack/kubearchive-install.sh

```diff
@@ -19,7 +19,7 @@ helm template kubearchive charts/kubearchive -n kubearchive \
     --include-crds \
     --set "global.production=true" > /tmp/kubearchive-not-resolved.yaml
 ko resolve -f /tmp/kubearchive-not-resolved.yaml --base-import-paths > ${YAML}
-kubectl apply -n kubearchive -f ${YAML}
+kubectl replace -n kubearchive -f ${YAML}
```

Comment:
I'm testing the PR locally and this does not work if I run the normal process, I think you changed this after installing for the first time. So this command can't `replace` if there are no resources. The process to debug another component needs to change to `hack/kubearchive-delete.sh` and then `hack/kubearchive-install.sh` again.

Reply:
You can see the error same error I got in the integration tests.

Reply:
Right, I tested just with some resources and see that even when they were not created with replace they were, but it seems that this doesn't apply for every resource in our script.

I've rollbacked the change in this file